### DISCUSSION TITLE
Intermediate check point for next step.

### DISCUSSION THREAD

**Discussion by rootwarp** [dsrvlabs/archway-cli#8] [General]
# 2022-05-16

## Current implementation status
- Init new local chain.
- Start local chain.
- Reset local chain.

## Remained issues
- Be able to snapshot and reset the chain state.
- Deploy and interact with contracts locally.

**Comment by rootwarp** [comment]
@Choi-Jinhong please leave your opinions here.

**Comment by rootwarp** [comment]
# Joonkyo
- `archway new` command don't allow `Local` environment for now. That should be enabled.

**Comment by gnongs** [comment]
## Jinhong
- `archway chain` command will be able to use `--home`.
- `archway chain stop` command will be add.
